From 7e1d84a0499cbd55afd4e80cffb1d2de74b44d74 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 4 Mar 2025 18:19:16 +0100
Subject: [PATCH 816/841] refactor(stm32mp21): remove CCB bindings

CCB peripheral is no more supported.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 5a562541a84370d4bc1d34e1c5c1d8fa15ec371d refactor(stm32mp21): remove CCB bindings]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: If2c963dcb389b61e141ecb060ad0fdc773fd237d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/441729
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 include/dt-bindings/clock/st,stm32mp21-rcc.h | 1 -
 include/dt-bindings/reset/st,stm32mp21-rcc.h | 1 -
 include/dt-bindings/soc/stm32mp21-rif.h      | 1 -
 3 files changed, 3 deletions(-)

diff --git a/include/dt-bindings/clock/st,stm32mp21-rcc.h b/include/dt-bindings/clock/st,stm32mp21-rcc.h
index 1b6d533e1..22b15f1d3 100644
--- a/include/dt-bindings/clock/st,stm32mp21-rcc.h
+++ b/include/dt-bindings/clock/st,stm32mp21-rcc.h
@@ -143,7 +143,6 @@
 #define CK_BUS_GPIOH		120
 #define CK_BUS_GPIOI		121
 #define CK_BUS_GPIOZ		122
-#define CK_BUS_CCB		123
 #define CK_BUS_RTC		124
 #define CK_BUS_LPUART1		125
 #define CK_BUS_LPTIM3		126
diff --git a/include/dt-bindings/reset/st,stm32mp21-rcc.h b/include/dt-bindings/reset/st,stm32mp21-rcc.h
index cacadf3fd..bf0e3a5d7 100644
--- a/include/dt-bindings/reset/st,stm32mp21-rcc.h
+++ b/include/dt-bindings/reset/st,stm32mp21-rcc.h
@@ -115,7 +115,6 @@
 #define HASH2_R		17440
 #define CRYP1_R		17472
 #define CRYP2_R		17504
-#define CCB_R		17536
 #define WWDG1_R		17696
 #define VREF_R		17760
 #define DTS_R		17792
diff --git a/include/dt-bindings/soc/stm32mp21-rif.h b/include/dt-bindings/soc/stm32mp21-rif.h
index baa034df4..e921edd9a 100644
--- a/include/dt-bindings/soc/stm32mp21-rif.h
+++ b/include/dt-bindings/soc/stm32mp21-rif.h
@@ -70,7 +70,6 @@
 #define STM32MP21_RIFSC_CSI_ID			86
 #define STM32MP21_RIFSC_DCMIPP_ID		87
 #define STM32MP21_RIFSC_DCMIPSSI_ID		88
-#define STM32MP21_RIFSC_CCB_ID			91
 #define STM32MP21_RIFSC_RNG1_ID			92
 #define STM32MP21_RIFSC_RNG2_ID			93
 #define STM32MP21_RIFSC_PKA_ID			94
-- 
2.47.2

